### PR TITLE
sinkv2(ticdc): add config to enable the new sink

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflow/issues/5928

### What is changed and how it works?

add config to enable the new sink.
#### Tests 

 - Unit test

##### Will it cause performance regression or break compatibility?
No
##### Do you need to update user documentation, design documentation or monitoring documentation?
No
### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/owner/ddl_sink.go [hunk 1]
```diff
@@ -103,11 +107,22 @@ type ddlSinkInitHandler func(ctx cdcContext.Context, a *ddlSinkImpl, id model.Ch
 func ddlSinkInitializer(ctx cdcContext.Context, a *ddlSinkImpl, id model.ChangeFeedID, info *model.ChangeFeedInfo) error {
 	stdCtx := contextutil.PutChangefeedIDInCtx(ctx, id)
 	stdCtx = contextutil.PutRoleInCtx(stdCtx, util.RoleOwner)
-	s, err := sink.New(stdCtx, id, info.SinkURI, info.Config, a.errCh)
-	if err != nil {
-		return errors.Trace(err)
+	conf := config.GetGlobalServerConfig()
+	if !conf.Debug.EnableNewSink {
+		log.Info("Try to create sinkV1")
```
Comment (hicqu) [inline cdc/owner/ddl_sink.go:112]: I prefer `create ddlSink based on sinkV1`.

Comment (hi-rustin) [reply cdc/owner/ddl_sink.go]: Done. Thanks!

**Review by hicqu — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

**Comment by hi-rustin** [comment]
We will try to add a new abstract to unify SinkV1 and SinkV2. Let's move forward.

/merge

**Pull request merged**